src/services/fs.ts

```diff
@@ -0,0 +1,15 @@
+import fs from "fs/promises"
+import path from "path"
+import prettier from "prettier"
```

Comment:
Is there a way to only include this file when `NODE_ENV=development`? Don't want these dips to be in runtime during prod execution.

Reply:
The PR is already doing that by importing `@services/fs` only if `isDev`